cyverse-de/sonora
# WIP: Support file uploads

Adds the API call for uploading files through Sonora and terrain and data-info and into the data-store.

Testing process (for now):
- Pull the PR with the `hub` tool.
- `npm install`
- `npm run dev` (assuming you have a local config).
- Hit http://localhost:3000/login in a browser to log in.
- Hit http://localhost:3000/upload to see the very basic test upload. Select a file and submit the form.
- See JSON in your browser.
- Check the data window in the current DE to make sure the file exists.

Currently the test upload form expects you to be logged in as ipcdev and will dump all of the files into ipcdev's home directory. That will change in the near future and this description will be modified to match.
